pkg/predicate/predicate.go

```diff
@@ -51,3 +58,38 @@ func (GenerationChangedPredicate) Update(e event.UpdateEvent) bool {
 	}
 	return true
 }
+
+// Skips events that have labels matching selectors defined in watches.yaml
+func (r ResourceFilterPredicate) eventFilter(eventLabels map[string]string) bool {
+	return r.Selector.Matches(labels.Set(eventLabels))
+}
+
+func NewResourceFilterPredicate(s metav1.LabelSelector) (ResourceFilterPredicate, error) {
+	// fmt.Println(s)
```

Comment:
These commented lines should be removed, and there probably doesn't need to be so many newlines (wouldn't normally even comment on that but since you'll be changing the file anyway figured I'd mention it)

Reply:
agreed on both counts